lib/urlwatch/reporters.py

```diff
@@ -161,37 +161,29 @@ def submit(self):
     def _parts(self):
         cfg = self.report.config['report']['html']
 
-        yield SafeHtml("""<!DOCTYPE html>
-        <html><head>
-            <title>urlwatch</title>
-            <meta http-equiv="content-type" content="text/html; charset=utf-8">
-            <meta name="viewport" content="width=device-width, initial-scale=1.0">
-            <style type="text/css">
-                body { font-family: sans-serif; }
-                .diff_add { color: green; background-color: lightgreen; }
-                .diff_sub { color: red; background-color: lightred; }
-                .diff_chg { color: orange; background-color: lightyellow; }
-                .unified_add { color: green; }
-                .unified_sub { color: red; }
-                .unified_nor { color: #333; }
-                table { font-family: monospace; }
-                h2 span.verb { color: #888; }
-            </style>
-        </head><body>
-        """)
+        yield SafeHtml(
+            '<!DOCTYPE html>\n'
+            '<html>\n'
+            '<head>\n'
+            '<title>urlwatch</title>\n'
+            '<meta http-equiv="content-type" content="text/html; charset=utf-8">\n'
+            '<meta name="viewport" content="width=device-width, initial-scale=1.0">\n'
+            # No stylesheet: styles are embedded to avoid overwriting by email clients
+            '</head>\n'
+            '<body>\n')
 
         for job_state in self.report.get_filtered_job_states(self.job_states):
             job = job_state.job
 
             if job.LOCATION_IS_URL:
                 title = '<a href="{location}">{pretty_name}</a>'
             elif job.pretty_name() != job.get_location():
-                title = '<span title="{location}">{pretty_name}</span>'
+                title = '<span title="{location}" style="color:#888888">{pretty_name}</span>'
             else:
                 title = '{location}'
-            title = '<h2><span class="verb">{verb}:</span> ' + title + '</h2>'
+            title = '<h2><span style="color:#888888">{verb}:</span> ' + title + '</h2>'
 
-            yield SafeHtml(title).format(verb=job_state.verb,
+            yield SafeHtml(title).format(verb=job_state.verb.title(),
                                          location=job.get_location(),
                                          pretty_name=job.pretty_name())
```

Comment:
In fact, here's an example how we use `str.format()` to separate the string contents from the formatting step (related to the question below regarding why `str.format()` is better than string concatenation a'la `' + ... + '`).